ansible_rulebook/app.py

```diff
@@ -282,7 +280,7 @@ def validate_actions(startup_args: StartupArgs) -> None:
                 if startup_args.check_vault:
                     for _key, value in action.action_args.items():
                         # attempt to substitute, raise an error on failure
-                        substitute_variables(value, {})
+                        decryptable(value)
```

Comment:
since decryptable can take a dict, do we still need this for loop?

Reply:
Changed